linalg/ode.hpp

```diff
@@ -24,9 +24,10 @@ class ODESolver
 protected:
    /// Pointer to the associated TimeDependentOperator.
    TimeDependentOperator *f;  // f(.,t) : R^n --> R^n
+   TimeDependentOperator *f2; // f(.,t) : R^n --> R^n
```

Comment:
Why do we need 2 of these now?

Reply:
This was added for using IMEX methods with ARKode.

Reply:
That makes sense.